components/brave_news/browser/feed_v2_builder.cc

```diff
@@ -123,10 +122,12 @@ ArticleWeight GetArticleWeight(const mojom::FeedItemMetadataPtr& article,
   const double source_visits_min = features::kBraveNewsSourceVisitsMin.Get();
   const double source_visits_projected =
       source_visits_min + signals.at(0)->visit_weight * (1 - source_visits_min);
-  const auto pop_recency = GetPopRecency(article);
+  const auto pop_recency =
+      base::ranges::min(GetPopRecency(article), 300.0) / 300.0;
```

Comment:
Okay, so we truncate now? I think it would be better to normalize this on the server but for now, it's definitely an improvement.

Reply:
Added https://github.com/brave/news-aggregator/issues/611

Reply:
Do we want to remove this and just let it get solved on the backend?

Also, maybe we should do the normalization before doing the recency stuff, to let the time have more of an effect? I haven't thought about it too much, but in https://github.com/brave/news-aggregator/issues/611 we're talking about normalizing the `pop_score` and here we're doing it to the `pop_recency`.

Reply:
Let's land without this. It should be solved on the backend. We can just assume that the popularity score will be between 0 and 100 as discussed in #661.